src/Bag.jl

```diff
@@ -0,0 +1,55 @@
+# This file is copied from YaoExtensions for convenience,
+# will be considerd to move to YaoBlocks!
+export AbstractBag
+"""
+    AbstractBag{BT, N}<:TagBlock{BT, N}
```

Comment:
Isn't this in YaoExtension or some early version of YaoBlocks already? it looks a bit messy now.

Reply:
yes，it turns out this wrapper class is pretty useful, it keeps all property of content (more strict than TagBlock). should be considered to moved back to YaoBlocks.